src/main/java/org/broadinstitute/hellbender/tools/funcotator/FilterFuncotations.java

```diff
@@ -143,12 +143,12 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
                 funcotationKeys, variant, "Gencode_" + reference.gencodeVersion + "_annotationTranscript", "FAKE_SOURCE");
 
         funcs.values().forEach(funcotationMap -> {
-            final Stream<Map<String, String>> transcriptFuncotations = funcotationMap.getTranscriptList().stream()
+            final Stream<Set<Map.Entry<String, String>>> transcriptFuncotations = funcotationMap.getTranscriptList().stream()
```

Comment:
I'm curious about the change to `Set` because has a wide impact on a lot of the code.

It seems fine, but I'm wondering about what prompted the refactor.